config/networks/testnet.json

```diff
@@ -73,18 +73,18 @@
       "committee_size": "0x5",
       "dag_blocks_size": "0xa",
       "ghost_path_move_back": "0x0",
-      "lambda_ms_min": "0x3e8",
+      "lambda_ms_min": "0x29a",
       "run_count_votes": false
     },
     "replay_protection_service": {
       "range": "0xa"
     },
     "vdf": {
-      "difficulty_max": "0x15",
+      "difficulty_max": "0x12",
       "difficulty_min": "0x10",
       "threshold_selection": "0xbffd",
       "threshold_vdf_omit": "0x6bf7",
-      "difficulty_stale": "0x16",
+      "difficulty_stale": "0x13",
```

Comment:
If only for test purpose and not permanence change, should not change here. Let's change it in config file